client/controller/index.web.js

```diff
@@ -4,36 +4,52 @@
 import React from 'react';
```

Comment:
Any reason for having controllers under a folder? I prefer to have the files directly in the root.